accounts-db/src/accounts_db.rs

```diff
@@ -3226,7 +3249,7 @@ impl AccountsDb {
                     let is_candidate_for_clean =
                         max_slot_inclusive >= *slot && latest_full_snapshot_slot >= *slot;
                     if is_candidate_for_clean {
-                        self.insert_pubkey(&candidates, *pubkey);
+                        insert_candidate(*pubkey, true);
```

Comment:
why is this 'true'?

Reply:
ah, because we are iterating `zero_lamport_accounts_to_purge_after_full_snapshot`

Reply:
yes. exactly!